client/src/index.css

```diff
@@ -370,11 +370,10 @@ outline: 1px solid slategrey;
     font-size: 25px;
 }
 }
-.anima:before
+.anima:before,.anima1:before,.anima2:before
```

Comment:
since these are going in homepage. please move these to the homepage.css. If we end up using this in other pages we can move back.